showcase/app/controllers/page-components/modal.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { deepTracked } from 'ember-deep-tracked';
+
+import type { PageComponentsModalModel } from 'showcase/routes/page-components/modal';
+
+export default class PageComponentsModalController extends Controller {
+  declare model: PageComponentsModalModel;
+
+  @deepTracked modals = {
+    basicModalActive: false,
+    longModalActive: false,
+    formModalActive: false,
+    tabsModalActive: false,
+    dropdownModalActive: false,
+    superselectModalActive1: false,
+    superselectModalActive2: false,
+    superselectModalActive3: false,
+    dismissDisabledModalActive: false,
+    dropdownInitiatedModalActive: false,
+    dropdownInitiatedWithReturnedFocusModalActive: false,
+    deactivateModalOnCloseActive: false,
+    deactivateModalOnDestroyActive: false,
+    deactivateModalOnSubmitActive: false,
+    deactivateModalOnSubmitValidationError: false,
```

Comment:
[todo] I don't think this should be in this list

Reply:
Agreed, I moved it back out.